### DISCUSSION TITLE
Allow a site to reference entries from a different site without enabling the section in that site

### DISCUSSION THREAD

**Discussion by gvjacob** [craftcms/cms#13559] [Ideas]
We are converting an existing Craft install to multisite. We have several sections that should be enabled for only one site and the primary site (or other sites) should be able to reference entries from that site (for example, in a Link field). Seems like this can be somewhat accomplished by setting the propagation method to `Save entries to other sites in the same group` and enabling the section for the primary site, although these entries won't inherit the right site's URI.

However, **we don't want to grant the primary site create/edit abilities for that section**, just the ability to reference them. I don't think this is currently possible but happy to be wrong! 😄 Otherwise, I think this is a compelling feature for the multisite paradigm. 

**Why is this important?**
Not all sites in a multisite network should be completely isolated from each other. In our case, we wanted to use multisite to better partition content, share sections between sites and allow for easier adjustments to entry types down the line, and cleanly configure user groups' permissions. It's very plausible for a site to want to reference content from a different one.

> Craft’s multi-site feature is not meant for managing multiple installs that have no relationship to one another, but different sites where it’s vital that some users and content are shared among them

Per the key selling point for multisite, this should help different sites share content among themselves.

**Comment by brandonkelly** [comment]
> for example, in a Link field

Which field type is that? Do you mean [Typed Link Field](https://plugins.craftcms.com/typedlinkfield?craft4)?

I’m not super familiar with the multi-site capabilities of third party plugins, but [Redactor](https://plugins.craftcms.com/redactor) and [CKEditor](https://plugins.craftcms.com/ckeditor) will each allow you to generate links to entries from other sites, and I can’t think of a reason that other plugins couldn’t do the same.
